examples/jenkins/README.md

```diff
@@ -10,40 +9,35 @@ the deployment works, and then tag the test version into production.
 Steps
 -----
 
-1. Follow steps 1-10 from the [sample-app](https://github.com/openshift/origin/blob/master/examples/sample-app/README.md)
+1. Follow steps 1-7 from the [sample-app](https://github.com/openshift/origin/blob/master/examples/sample-app/README.md)
 
-2. Start the Jenkins services
+2. Start the Jenkins service
 
-        $ oc create -n test -f jenkins-config.json
+        $ oc new-app jenkins-ephemeral-template.json
         
-3. Create the application configuration
+    Note: This template uses an EmptyDir type volume.  If you want to ensure your jenkins configuration/job information is persisted through pod restarts and deployments, you will want to use a [PersistentVolume](https://docs.openshift.org/latest/admin_guide/persistent_storage_nfs.html) instead.
+
+3. Create the sample application configuration
 
-        $ oc process -n test -f application-template.json | oc create -n test -f -
+        $ oc new-app application-template.json
  
 4. Locate the Jenkins service endpoint and go to it in your browser:
 
         $ oc get services -n test jenkins --template="{{ .spec.portalIP }}:{{ with index .spec.ports 0 }}{{ .port }}{{ end }}"
 
     Once it is available, proceed to the next step.
-    
-5. Create the Jenkins job named rubyJob:
+   
+5. Select the the `OpenShift Sample` job and click `Configure`
 
-        $ JENKINS_ENDPOINT=`oc get services -n test jenkins --template="{{ .spec.portalIP }}:{{ with index .spec.ports 0 }}{{ .port }}{{ end }}"`
-        $ cat job.xml | curl -X POST -H "Content-Type: application/xml" -H "Expect: " --data-binary @- http://$JENKINS_ENDPOINT/createItem?name=rubyJob
+6. Retrieve your credentials so you can supply them to Jenkins:
 
-6. Add API credentials to the Jenkins build:
+        $ oc config view --minify --flatten
```

Comment:
after talking to @deads2k i decided to abandon my lofty goals of using service accounts for this.  it seemed to just make the instructions to the user significantly more complex while not really buying anything.